Tests/SentryTests/Protocol/SentryUserFeedbackTests.swift

```diff
@@ -18,19 +19,19 @@ class SentryUserFeedbackTests: XCTestCase {
         
         let actual = userFeedback.serialize()
         
-        XCTAssertEqual(userFeedback.eventId.sentryIdString, actual["event_id"] as? String)
```

Comment:
`as? String` might not be necessary, assuming Nimble checks if the types match?

Reply:
The actual is of type `[String:Any]`. That's why you need to cast it. It didn't compile without `as ? String`. If you know how to do it without, please tell me.

Reply:
no was just a hunch - I guess if UserFeedback is a bridged ObjC type it won't work - no need to invest more time